packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -9,6 +9,7 @@ import {
   AccessControlRegistryFactory,
```

Comment:
I can't comment on it but there's an empty file above `packages/airnode-admin/src/utils.ts` ?

Reply:
Thank you, I'll remove it :small_airplane: